cmd/kar-controllers/app/options/options.go

```diff
@@ -30,6 +30,10 @@ type ServerOption struct {
 	Dispatcher	bool
 	AgentConfigs 	string
 	SecurePort	int
+	DynamicPriority	bool  // If DynamicPriority=true then no preemption is allowed by program logic
+	Preemption 		bool  // Preemption is not allowed under DynamicPriority
```

Comment:
Could `DynamicPriority` be used to toggle `Preemption` instead of having the `Preemption` parameter?  Is there a use case (policy) we want to support for various combinations of these parameters, e.g. `Preemption=true` && `DynamicPriority=true`?

Reply:
Strictly speaking, it is possible to have all 4 possible combinations: Preemption=true or false, DynamicPriority=true or false.  However, having Preemption=true && DynamicPriority=true is generally not preferred.  So the implementation disables this combination while allowing all 3 other combinations.

Reply:
Ok, I think the proper approach is to parameterize each policy based on applicable options.  I will open an addition issue address this enhancement.  I do suggest that the settings be updated to reflect the controller behavior.   Resolving this conversation and per opening an issue as described above and addressing the 4th condition in the parameter settings.  Thanks!